php/commands/secure.php

```diff
@@ -36,11 +36,60 @@ public function secure_port(){
 		//***This function Secures port***
 		//todo:
 
+		EE::info("Please Enter valid port number");
+		$port = EE::input_value("EasyEngine admin port [22222]:");
+		if(empty($port)){
+			EE::debug("No port selected using default port : 22222");
+			$port = 22222;
+		}
+		
+		while(!is_numeric($port)){
+			EE::debug("Not a valid port");
+			EE::error("Please enter valid port number:");
+			$port = EE::input_value("EasyEngine admin port [22222]:");
+		}
+		EE::debug("Checking distro of the machine");
+		$distro_name = EE_OS::ee_platform_distro();
+		EE::debug("$distro_name found");
+		if("ubuntu" == $distro_name) {
+			EE::debug("Changing nginx configuration");
+			EE::exec_cmd("sed -i \"s/listen.*/listen $port default_server ssl http2;/\" /etc/nginx/sites-available/22222");
+		} else if ( "debian" == $distro_name){
+			EE::debug("Changing nginx configuration");
+			EE::exec_cmd("sed -i \"s/listen.*/listen $port default_server ssl http2;/\" /etc/nginx/sites-available/22222");
+		} else {
+			EE::error("Unknown enviorment found");
+		}
+		EE::debug("Adding changes to git");
+		EE_Git::add("/etc/nginx","Adding changed port to git");
+		EE::debug("Restarting nginx");
+		EE_Service::restart_service("nginx");
 	}
 
 	public static function secure_ip(){
 		//***This function Secures IP***
 		//todo:
+		$newiplist = array();
```

Comment:
@bnap00 
I think you are not using this var can you please remove this one if not ?